operations/experimental/enable-nfs-broker-backup.yml

```diff
@@ -0,0 +1,14 @@
+---
+  - type: replace
+    path: /instance_groups/name=backup-restore/jobs/-
+    value:
+      name: nfsbroker-bbr
+      release: nfs-volume
+      properties:
+        nfsbroker:
+          db_driver: ((nfsbroker-db-driver))
+          db_username: ((nfsbroker-db-username))
```

Comment:
This is currently set to `nfs-broker` in `enable-volume-service-nfs.yml`: https://github.com/cloudfoundry/cf-deployment/blob/master/operations/community/enable-volume-service-nfs.yml#L71